src/devices/SoftPwm/SoftwarePwmChannel.cs

```diff
@@ -103,9 +105,10 @@ private void UpdateRange()
 
         private void RunSoftPWM()
         {
-            if (_precisionPWM)
+            if (_usePrecisionTimer)
             {
                 Thread.CurrentThread.Priority = ThreadPriority.Highest;
+                System.Interop.ThreadHelper.SetCurrentThreadHighPriority();
```

Comment:
This won't work on Windows right? If we want we can check here where are we running and if running in Linux we can then call this method. This is actually probably the reason why you needed to set a runtimeIdentifier before so let's just put it inside an if check that checks the runtimeinformation to see if we are in Linux.

Reply:
other options is perhaps just remove this for now and we can address this separately (we might wanna do similar fix for LED matrix as well)

Reply:
Removed in last commit